src/sentry/receivers/sentry_apps.py

```diff
@@ -118,7 +118,7 @@ def send_comment_deleted_webhook(project, user, group, data, **kwargs):
 def send_comment_webhooks(organization, issue, user, event, data=None):
     data = data or {}
 
-    for install in installations_to_notify(organization, event):
+    for install in installations_to_notify(organization, "comment"):
```

Comment:
@sentaur-athena so we don't need `event` anymore?

Reply:
yes, that's the idea. To know which installations we need to notify, we need to know which ones subscribed to any event in this category. In this case the category for all events is "comment" so I didn't get it from event. I can replace it with event.split(".")[0]